config/manager/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+resources:
+- manager.yaml
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+- files:
+  - controller_manager_config.yaml
+  name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: 667741767953.dkr.ecr.us-west-2.amazonaws.com/vivek/splunk/splunk-operator
```

Comment:
this will automatically change as this is passed as command line argument to make : example 
> make docker-build docker-push IMG=docker.io/splunk/splunk-operator:1.1.0 
so it will show 
images:
- name: controller
  newName: docker.io/splunk/splunk-operator
  newTag: 1.1.0

Reply:
fixed